src/mixins/atoms.js

```diff
@@ -71,18 +77,111 @@ export const AtomsMixin = (superclass) => class extends superclass {
     createAtomsGroup(basis, atomRadiiScale) {
         const atomsGroup = new THREE.Group();
         atomsGroup.name = "Atoms";
-        atomsGroup.add(...basis.coordinates.map((atomicCoordinate, atomicIndex) => {
+        basis.coordinates.forEach((atomicCoordinate, atomicIndex) => {
             const element = basis.getElementByIndex(atomicIndex);
             const sphereMesh = this.getSphereMeshObject({
                 ...this._getDefaultSettingsForElement(element, atomRadiiScale),
                 coordinate: atomicCoordinate.value,
             });
             sphereMesh.name = `${element}-${atomicIndex}`;
-            return sphereMesh;
-        }));
+            atomsGroup.add(sphereMesh);
+        });
         return atomsGroup;
     }
 
+    /**
+     * Whether to draw a bond between given elements.
+     */
+    areElementsBonded(element1, coordinate1, element2, coordinate2) {
+        return Boolean(ELEMENT_BONDS.find(b => {
+            return b.elements.includes(element1) && b.elements.includes(element2) &&
+                b.length.value && b.length.value >= Made.math.vDist(coordinate1.value, coordinate2.value);
+        }));
+    }
+
+    /**
+     * Add bonds.
```

Comment:
The docstring is not adding any information - either elaborate the logic or remove